lib/customer.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/menu'
+
+class Customer
+  def place_order(dish, quantity)
```

Comment:
Unused method argument - `dish`. If it's necessary, use `_` or `_dish` as an argument name to indicate that it won't be used. You can also write as `place_order(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. You can also write as `place_order(*)` if you want the method to accept any arguments but don't care about them.